tests/installation/ipxe_install.pm

```diff
@@ -70,11 +70,19 @@ sub set_bootscript {
     # trim all strings from variables to get rid of bogus whitespaces
     $url =~ s/^\s+|\s+$//g;
     $arch =~ s/^\s+|\s+$//g;
-    $autoyast =~ s/^\s+|\s+$//g;
-    $regurl =~ s/^\s+|\s+$//g;
-    $console =~ s/^\s+|\s+$//g;
+    $autoyast =~ s/^\s+|\s+$//g if $autoyast;
+    $regurl =~ s/^\s+|\s+$//g if $regurl;
+    $console =~ s/^\s+|\s+$//g if $console;
```

Comment:
I do not think these three ```if``` are very necessary. Keep these line as what they were is enough.

Reply:
They bring `uninitialized value ...` warnings in auto-inst.txt. To remove this kind of warning and make logs clean follows https://progress.opensuse.org/issues/120750.

Reply:
```$autoyast``` already initialized by ```get_var('AUTOYAST', '');```. So I think you just need
```
my $regurl = get_var('SCC_URL', '');
my $console = get_var('IPXE_CONSOLE', '');
```
instead of using ```if```.

Reply:
Yes, it works. thanks.